### DISCUSSION TITLE
How to move a window into split mode?

### DISCUSSION THREAD

**Discussion by Today20092** [glzr-io/glazewm#580] [Q&A]
I'm trying to figure out how to split the window like this. I'm not sure how to do it.
![image](https://github.com/glzr-io/glazewm/assets/79241611/445fab69-371b-4cd4-95f0-06cafe8acea1)

**Comment by aaronedev** [comment]
alt+v

**Comment by MyGitHubBlueberry** [comment]
Open first window -> open second window -> switch layout mode to vertical (Alt-v) -> open third window
